### PR TITLE
Update the de_metas_endcustomer_fresh_reports.hasFreeEUTaxes Function

### PR DESCRIPTION
**Pull request by riadmen**
https://github.com/metasfresh/metasfresh/issues/16164

### CODE REVIEW

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5701720_sys_gh16164_C_Invoice_V.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
+CREATE VIEW de_metas_endcustomer_fresh_reports.c_invoice_v
+            (
+             value,
```
Comment (cristinamghita) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5701720_sys_gh16164_C_Invoice_V.sql:3]: Why are you adjusting this?

**Review by cristinamghita — CHANGES_REQUESTED** [review]
See my question.

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5701720_sys_gh16164_C_Invoice_V.sql (hunk 1 above)
Comment (riadmen) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5701720_sys_gh16164_C_Invoice_V.sql:3]: `c_invoice_v` depend on `de_metas_endcustomer_fresh_reports.hasFreeEUTaxes` so i need to drop this view before updating the function, and then create it egain

Comment (cristinamghita) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5701720_sys_gh16164_C_Invoice_V.sql:3]: @riadmen but there is no drop. You just recreate the view.
Pls, do that is the same file with function:
Drop the view, drop the function, recreate the function, recreate the view.

**Review by cristinamghita — CHANGES_REQUESTED** [review]
Check my comment

**Review by cristinamghita — APPROVED** [review]
lgtm

**Pull request merged**